bark/world/world.cpp

```diff
@@ -244,6 +244,11 @@ FrontRearAgents World::GetAgentFrontRearForId(
     }
 
     FrenetPosition frenet_other(it->second->GetCurrentPosition(), center_line);
+    float width = lane_corridor->GetLaneWidth(it->second->GetCurrentPosition());
+    if (abs(frenet_other.lat) > width / 2) {
```

Comment:
use std::abs
see: https://stackoverflow.com/questions/33738509/whats-the-difference-between-abs-and-fabs